db/schema.rb

```diff
@@ -171,7 +173,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.uuid "school_id", null: false
     t.uuid "lead_provider_id", null: false
-    t.uuid "cohort_id", default: "f85b3d42-9f6e-41e2-a014-0a09caea7e93", null: false
+    t.uuid "cohort_id", default: "d97d273b-7519-4b1b-a62f-a12c39c4b98c", null: false
```

Comment:
I think we should remove the default cohort id - or in every pull request we will see a different id from each one of us after changing db

Reply:
Happy for this to be a new PR? https://github.com/DFE-Digital/early-careers-framework/pull/112#discussion_r589345975